temporal/t.upgrade/t.upgrade.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+############################################################################
+#
+# MODULE:       t.upgrade
+# AUTHOR(S):    Martin Landa, Markus Neteler
+#
+# PURPOSE:      Upgrade of TGRASS DB
+# COPYRIGHT:    (C) 2019 by Martin Landa, and the GRASS Development Team
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#############################################################################
+
+# %module
+# % description: Upgrades the version of a space time dataset (TGRASS DB).
+# % keyword: temporal
+# % keyword: metadata
+# % keyword: extent
+# % keyword: time
+# %end
+
+import sys
+import grass.script as grass
+
+
+def main():
+    # lazy imports
+    import grass.temporal as tgis
+
+    tgis.init(skip_db_version_check=True)
+
+    dbif = tgis.SQLDatabaseInterfaceConnection()
+    dbif.connect()
+
+    tgis.upgrade_temporal_database(dbif)
+
+    return 0
+
+
+if __name__ == "__main__":
+    options, flags = grass.parser()
+    sys.exit(main())
```

Comment:
This module can be even simpler: `return 0` and implicit `return None` have the same effect. I don't think there is a need even for `sys.exit()`. `options, flags` don't need to be global, it is just enough to call `parser()` before the lazy import. `import grass.script as grass` is really old style which likely emerged  during an early transition from single package to subpackages. Documentation suggests `import grass.script as gs`. Also, contrary to the popular belief, Python doesn't require an empty line between two statements. See e.g. simplifications done in the r.centroids PR in addons for similar changes.